pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -435,9 +461,21 @@ func flowMergeProcess(entry *netlink.ConntrackFlow, instData networkAttrs) flowS
 	return ipFlow
 }
 
-func checkAppIPAddr(appAddrs []net.IP, entryIP net.IP) bool {
-	for _, appIP := range appAddrs {
-		if strings.Compare(appIP.String(), entryIP.String()) == 0 {
+func flowGetAppInfo(tuple flowStats, appinformation []appInfo) appInfo {
+	var appinfo appInfo
+	for _, info := range appinformation {
+		if strings.Compare(info.ipaddr.String(), tuple.SrcIP.String()) == 0 ||
```

Comment:
Note that you can do ip1.Equal(ip2) and avoid the string conversion and compare.

Reply:
ok